src/cmds/scripts/pbs_snapshot

```diff
@@ -38,14 +38,24 @@
 ptllibpath=
 ptlbinpath=
 
+# Try rpm install paths
 if [ -f "/etc/debian_version" ]; then
-        ptl_prefix_lib=$( dpkg -L pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null )
+        ptl_prefix_lib=$(dpkg -L pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null)
 else
-        ptl_prefix_lib=$( rpm -ql pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null )
+        ptl_prefix_lib=$(rpm -ql pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null)
 fi
+
+# Try system paths
+if [ ! -d  ${ptl_prefix_lib} ]; then
+    ptl_prefix_lib=/usr/local/lib/python3.[[:digit:]]/site-packages
+    if [ ! -d  ${ptl_prefix_lib} ]; then
+        ptl_prefix_lib=/usr/lib/python3.[[:digit:]]/site-packages
```

Comment:
Can ls take [[:digit:]], I thought it can only take globs.

Reply:
Seems to work:

[ravi@pbspro ~]$ ptl_prefix_lib=/usr/lib/python3.[[:digit:]]/site-packages
[ravi@pbspro ~]$ echo $ptl_prefix_lib
/usr/lib/python3.6/site-packages
[ravi@pbspro ~]$ ls $ptl_prefix_lib
easy_install.py  pip  pip-9.0.3.dist-info  pkg_resources  __pycache__  setuptools  setuptools-39.2.0.dist-info
[ravi@pbspro ~]$

Reply:
This is interesting.  I wouldn't have thought it would work, but from the looks of it, the regex is being evaluated at the assignment, not at ls.  When it hits ls, it is a path.

Reply:
Looks like it works for bash, but would probably fail in Bourne/Korn/C shell.

Reply:
Thanks for your input Mike. It seems to work on Korn and Bourne as well, but it failed on C shell.

Reply:
Good to know. Thanks @agrawalravi90!